profile/README.md

```diff
@@ -0,0 +1,33 @@
+Hi, fellow open source lovers 👋
+
+We’re extremely excited that you landed on our GitHub page!
+
+At Mondoo, we’re staunch proponents of open source and all the progress that has been made in our industry because of it. A hearty thank you is in order to the community, project maintainers, and contributors. Your efforts have made people’s lives easier ❤️
```

Comment:
This reads a little weird to me. It talks about all the thanks to the communities/maintainers/contributors. Then it says `your efforts`. That can be read as if the efforts are of the person who landed at this page (who may or may not have any involvement in open source), and it makes more sense to me to say something like `All these efforts have made...`

Reply:
I also have some concerns about this bit. Any ideas on a replacement?